### PR TITLE
Added clusterID at metrics labels

### PR DESCRIPTION
**Pull request by rafael-azevedo**
This PR adds clusterID to RMO metrics per the  osd ticket https://issues.redhat.com/browse/OSD-6726

### CODE REVIEW

**Comment by georgettica** [comment]
@rafael-azevedo did you check the teat-integration?

Aside of that it looks ok

File: pkg/blackboxexporter/blackboxexporter.go [hunk 1]
```diff
@@ -12,6 +11,7 @@ import (
 	"k8s.io/apimachinery/pkg/util/intstr"
 	"sigs.k8s.io/controller-runtime/pkg/client"
 
+	blackboxConst "github.com/openshift/route-monitor-operator/pkg/consts/blackboxexporter"
```
Comment (georgettica) [inline pkg/blackboxexporter/blackboxexporter.go:14]: Why did you make the package uppercase?
```suggestion
	blackboxconst "github.com/openshift/route-monitor-operator/pkg/consts/blackboxexporter"
```

**Comment by ramonbutter** [comment]
With having [PR 110](https://github.com/openshift/route-monitor-operator/pull/110) merged, this is not required anymore and can be closed as the changes are already included. Sorry that I messed it up.

**Comment by georgettica** [comment]
This can be closed from what I see, as it's on master already from a different PR

**Review by NautiluX — COMMENTED** [review]
/close
done in #110

**Pull request closed without merging**